amqcfg/profiles/artemis/2.14.0/store/jdbc/default.yaml

```diff
@@ -0,0 +1,77 @@
+---
+
+_defaults:
+  broker_name: amq
+  broker_home: /opt/amq-broker-7.4.0
```

Comment:
IDK how you want it, but if deploying artemis, maybe you want to change broker_home to /opt/artemis instead?
My old automation used amq-broker* but imo it should not be there...